errors/error.go

```diff
@@ -48,6 +54,16 @@ func New(e interface{}, skip int) *Error {
 			Err:   e,
 			stack: e.Callers(),
 		}
+	case errorWithStack:
+		trace := e.StackTrace()
+		stack := make([]uintptr, len(trace))
+		for i, ptr := range trace {
+			stack[i] = uintptr(ptr)
```

Comment:
Should this be

```suggestion
			stack[i] = uintptr(ptr) - 1
```
?

[The docs say:](https://pkg.go.dev/github.com/pkg/errors#Frame)
> For historical reasons if Frame is interpreted as a uintptr its value represents the program counter + 1. 

Seems like the test is working though, so I'm not sure which is correct

Reply:
I tried both ways and somehow it didn't break! 🤔 I'm going to make the change though because I meant to do it but then forgot when the tests worked.